pages/api/models.ts

```diff
@@ -18,16 +18,28 @@ export const config = {
 };
 
 const getModels = (json: any, configData: any) => {
-  return json.data
+  return json.data.filter((model: any) => {
+    if (model?.capabilities) {
+      return model.capabilities.chat_completion;
+    } else {
+      return true;
+    }
+  })
       .map((model: any) => {
-        const model_name = (configData.OPENAI_API_TYPE === 'azure') ? model.model : model.id;
+        let model_passed = true;
+        const model_name = (configData.OPENAI_API_TYPE === 'azure' && model.model) ? model.model : model.id;
```

Comment:
They changed the data structure of models to (more closely?) match openai's output. This accounts for that, but allows some level of backwards compatibility.